cacio-tta/pom.xml

```diff
@@ -45,6 +45,17 @@
       <artifactId>jide-oss</artifactId>
       <version>3.6.18</version>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/net.bytebuddy/byte-buddy -->
+    <dependency>
+      <groupId>net.bytebuddy</groupId>
+      <artifactId>byte-buddy</artifactId>
+      <version>1.14.11</version>
+    </dependency>
+    <dependency>
+      <groupId>net.bytebuddy</groupId>
+      <artifactId>byte-buddy-agent</artifactId>
+      <version>1.14.11</version>
+    </dependency>
```

Comment:
I'm wondering if these should have a `provided` scope or perhaps shadow the jar.  In my case I'll want to use whichever version the Spring Boot bom defines. I'll leave this for now but might consider that change later.

Reply:
I am less familiar with Spring Boot and boms. I generally go for the simplest thing that works and makes sense to me.